rules/backlog-refinement-meeting/rule.md

```diff
@@ -62,13 +62,14 @@ Product Backlog: {{ LINK TO PRODUCT BACKLOG }}
 #### Agenda
 
 1. Skip all PBIs that are already marked as "Ready"
-2. Refine and estimate the top 5 PBIs that are not marked as "Ready" or in the "Not Ready" state
+2. Refine and estimate the top PBIs that are not marked as "Ready" or in the "Not Ready" state
+    * You should aim to have enough ready PBIs to cover work for the next 2-3 Sprints

```

Comment:
I just timebox it - spend an hour refining each week.

That way the meeting has a boundary and you are limited x PBIs